frontend/src/pages/Account.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import '../styles/App.scss';
+import { updateMe } from '../actions/users';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchMe } from '../actions/users';
+
+//Account page for update user's info
+class AccountPage extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.onChangeEmail = this.onChangeEmail.bind(this);
+    this.onChangeNickName = this.onChangeNickName.bind(this);
+    this.state = { showresult: false };
+  }
+
+  componentDidMount() {
+    this.props.fetchMe();
+  }
+
+  onChangeEmail(e) {
+    const state = this.props.user;
+    state[e.target.name] = e.target.value;
+    this.setState(state);
+  }
+
+  onChangeNickName(e) {
+    const state = this.props.user;
+    state[e.target.name] = e.target.value;
+    this.setState(state);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const data = this.state;
+    this.setState({ showresult: true });
+    this.props.updateMe(data);
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className="row">
+          <div className="col-md-9">
+            <h4>My Account</h4>
+            <br />
+            <form className="form-horizontal" onSubmit={this.handleSubmit}>
+              <div className="form-group">
+                <label className="col-lg-3 control-label">Nickname:</label>
+                <div className="col-lg-8">
+                  <input
+                    className="form-control"
+                    placeholder="Nickname"
+                    name="nickname"
+                    type="text"
+                    defaultValue={
```

Comment:
this should be `value`, and should just be `value={this.props.user.me.nickname}`, as you initialized nickname to empty string inside reducer_user